webclient/src/components/elements/ValidatedTextBox.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+    <div class = "input">
+        <span class='input-label'>{{fieldLabel}}</span>
+        <input type='text' :placeholder='placeholder' v-model="model" v-validate.initial = "validateExpression" 
```

Comment:
With `type='text'` hardcoded the password input box now shows the password entered. Can you please correct that?